### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by jicsontoh**

### CODE REVIEW

**Review by Jo3LW — APPROVED** [review]
LGTM!

**Review by ElginL — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -958,6 +958,16 @@ testers are expected to do more *exploratory* testing.
    1. Prerequisites: List all clients using the `list` command. More than one client in the list.
    2. Test case: `buy 1 g/apples price/test q/100`<br/>Expected: No transaction is added. Error details shown in the Application's Reply panel.
 
+### Help Command
+
+1. Executing the Help Command.
+
+    1. Test case: `help`<br/>Expected: Display all the valid commands' usage message.
+    2. Test case: `help buy`<br/>Expected: Display help commands usage message.
+    3. Test case: `help me please`<br/>Expected: Not a valid command. Error details shown in the `Application's Reply` panel.
```
Comment (notle1706) [inline docs/DeveloperGuide.md:967]: Perhaps there should be an ' here

**Review by notle1706 — APPROVED** [review]
LGTM!

**Pull request merged**